analytics-listeners/src/main/java/org/exoplatform/analytics/listener/tasks/TaskSavedListener.java

```diff
@@ -119,32 +119,21 @@ private String computeTaskOperation(TaskDto oldTask, TaskDto newTask) {
     if (oldTask != null) {
       if (isDiff(oldTask.isCompleted(), newTask.isCompleted())) {
         taskOperation = "taskCompleted";
-      } else if (isDiff(oldTask.getDescription(), newTask.getDescription())) {
-        taskOperation = "taskDescriptionChanged";
-      } else if (isDiff(oldTask.getTitle(), newTask.getTitle())) {
-        taskOperation = "taskTitleChanged";
+      } else if (isDiff(oldTask.getDescription(), newTask.getDescription()) || isDiff(oldTask.getTitle(), newTask.getTitle()) || isDiff(oldTask.getPriority(), newTask.getPriority())
+                 || isDiff(oldTask.getRank(), newTask.getRank()) || isDiff(oldTask.getAssignee(), newTask.getAssignee()) || isDiff(oldTask.getCoworker(), newTask.getCoworker())) {
+        taskOperation = "taskUpdated";
       } else if (isDiff(oldTask.getDueDate(), newTask.getDueDate())) {
-        taskOperation = "taskDueDateChanged";
-      } else if (isDiff(oldTask.getPriority(), newTask.getPriority())) {
-        taskOperation = "taskPriorityChanged";
-      } else if (isDiff(oldTask.getRank(), newTask.getRank())) {
-        taskOperation = "taskRankChanged";
-      } else if (isDiff(oldTask.getAssignee(), newTask.getAssignee())) {
-        taskOperation = "taskAssigneeChanged";
-      } else if (isDiff(oldTask.getCoworker(), newTask.getCoworker())) {
-        taskOperation = "taskCoworderChanged";
```

Comment:
it's better to not reduce granularity and change instead the configured parameters in parameters to display those operations instead.